assets/ui/environmentInfoScreen.ui

```diff
@@ -0,0 +1,58 @@
+{
+    "type": "EnvironmentInfoScreen",
```

Comment:
This should be called something other than `EnvironmentInfoScreen`, though 🙂 It conflicts with the screen in `TutorialNui`, and other students who work on this task later might also run into conflicts.